docs/hub/datasets-image.md

```diff
@@ -0,0 +1,123 @@
+# Image Dataset
+
+This guide will show you how to configure your dataset repository with image files.
+
+A dataset with a supported structure and [file formats](./datasets-adding#file-formats) automatically has a Dataset Viewer on its dataset page on the Hub. Any additional information about your dataset - such as captioning or object detection - is automatically loaded as long as you include this information in a metadata file (`metadata.csv`/`metadata.jsonl`).
+
+## Only images
+
+If your dataset only consists of one column with images, you can use a simple setup: store your image files at the root:
+
+```
+1.jpg
+2.jpg
+3.jpg
+4.jpg
+```
+
+or using subdirectories:
```

Comment:
note that if there is more than one subdirectory, their names are by default treated as class names and a `label` column is added, as described in "Image classification" section below (this behavior can be changed by setting the `drop_label` parameter in readme's yaml but this is not relevant to this section). 
so maybe:
```suggestion
or in a subdirectory:
```

Reply:
Ohhhh you're right. Changed